tutorials/abap-custom-ui-trust-cf/abap-custom-ui-trust-cf.md

```diff
@@ -0,0 +1,163 @@
+
+**Additional Information**
+>Tutorial last updated in May 2023
+
+- **Documentation:** [SAP Cloud Identity Services - Identity Authentication](https://help.sap.com/viewer/6d6d63354d1242d185ab4830fc04feb1/Cloud/en-US/d17a116432d24470930ebea41977a888.html)
+
+>Be aware that in case of an integration with SAP S/4HANA Cloud the used Identity Authentication for the subaccount should be the very same as the one used for the SAP S/4HANA Cloud system.
+>
+>Your SAP S/4HANA Cloud system you got already delivered by SAP with a configured trust between it and your SAP Cloud Identity Services tenant. Now you will configure the trust between that and your SAP Business Technology Platform subaccount on your own.
+>
+>![SAP S/4HANA Cloud and SAP Business Technology Platform subaccount share same Identity Provider](trust_IAS_SCP.png)
+
+
+---
+<!--tested with https://canary.cockpit.btp.int.sap/cockpit/#/globalaccount/2fcd6ac6-b8e0-40e8-aa71-a357aa99585e/subaccount/f57f211e-2733-4cc6-b645-74f02d034a58/subaccountoverview (ABAP Environment Staging Consump -> Integration Tests Staging (CF))-->
+
+### Get SAML metadata of SAP BTP subaccount
+
+To set up the trust from Identity Authentication to the subaccount you need the subaccount's SAML metadata.
+
+![Enter SAP BTP Trust Configuration and get metadata](btp-open-trust-config-get-metadata.png)
+
+1. Enter the SAP Business Technology Platform subaccount as an administrator and expand the **Security** area.
+   
+2. Open **Trust Configuration**.
+   
+3. Click **SAML Metadata**.
+
+
+### Enter SAP Cloud Identity Services administration console
+
+Open the SAP Cloud Identity Services administration console with its URL which follows the pattern:
+
+`https://<YOUR_TENANTS_ID>.accounts.ondemand.com/admin`
+
+The Tenant ID is an automatically generated ID by the system. The first administrator created for the tenant receives an activation e-mail with an URL in it. This URL contains the tenant ID.
+
+SAP Cloud Identity Services administration console entry screen looks (depending on authorizations) like that
+
+![Enter SAP Cloud Identity Services administration console](IAS_entryScreen.png)
+
+
+### Add SAP BTP subaccount as application
+
+The subaccount is represented in SAP Cloud Identity Services as Application.
+
+Choose **Applications & Resources** (1) and go to **Applications** (2). Click **Create** (3) on the left hand panel and enter a **Display Name** (4) to represent your SAP Business Technology Platform subaccount. **Save** (5) your changes.
+
+![Add subaccount as application](IAS_addApplication.png)
+
+
+### Configure application's trust with subaccount
+
+1. The newly created application will be shown, choose **SAML 2.0 Configuration**.
+
+    ![Configure application' s SAML 2.0 trust with subaccount](IAS_openSamlConfig.png)
+
+2. **Browse** for the metadata XML file of your SAP Business Technology Platform subaccount and upload it. By this all needed properties are gotten from the XML file.
```

Comment:
This is the first time that you mention the XML file in the tutorial. I assume you mean the SAML metadata that you go in step 1. I would suggest to referring to it with a consistent name throughout the tutorial, to avoid confusion. You can even explicitly refer to the step where you got it, e.g.: '**Browse** for the metadata XML file of your SAP Business Technology Platform subaccount that you downloaded in Step 1, and upload it'